rmi/servercode/ResImpl/ResourceManagerImpl.java

```diff
@@ -18,7 +18,8 @@
 public class ResourceManagerImpl implements ResourceManager
 {
 

```

Comment:
the RMHashtable is now public and synchronized, might need to tighten that up later and write some private methods, idk